BryantUniversity/BryantUniversity/Controllers/HomeController.cs

```diff
@@ -1,12 +1,44 @@
-﻿using System.Web.Mvc;
+﻿using BryantUniversity.Data;
+using BryantUniversity.Models.Repo;
+using BryantUniversity.ViewModels;
+using System.Web.Mvc;
 
 namespace BryantUniversity.Controllers
 {
     [AllowAnonymous]
     public class HomeController : Controller
     {
-        // GET: Home
-        public ActionResult Index()
+        private Context context = new Context();
+        [AllowAnonymous]
+        public ActionResult Calendar()
+        {
+            var viewModel = new SemesterDetailsViewModel();
+            CalendarRepo cRepo = new CalendarRepo(context);
```

Comment:
Whenever you use `context`, you should wrap it in a `using` block to make sure that it gets disposed correctly.

Reply:
hm, I think I forgot how that would look, would it be something like 
using(context){
       CalendarRepo cRepo = new CalendarRepo(context);
}
?

Reply:
Yes. If `context` wasn't already instantiated in the constructor, you could also do something like:
```
using (var context = new Context()) 
{
    // use context in this block
}
```

If you set up dependency injection, you wouldn't need to dispose of `context`; the DI container would take care of that for you.